### PR TITLE
feat: add logger middleware for dev/local environment

### LINKED ISSUE #37: feat: add logger middleware for dev/local environment
Reference: https://docs.gofiber.io/api/middleware/logger/

### PR DESCRIPTION
**Pull request by martinyonatann**
Closes #37

### CODE REVIEW

**Comment by championswimmer** [comment]
@martinyonatann 
can you take a pull and redo go get so the go.sum is sorted out again?

File: src/server/server.go [hunk 1]
```diff
@@ -29,6 +31,7 @@ import (
 //	@security					API Key
 func CreateApp(db *gorm.DB) *fiber.App {
 	app := fiber.New()
+	app.Use(logger.NewLogger())
```
Comment (championswimmer) [inline src/server/server.go:34]: for now this is hardcoded to run even on prod right?

Comment (martinyonatann) [reply src/server/server.go:34]: yes is it, next we should discuss about log format.

Comment (championswimmer) [reply src/server/server.go:35]: can we do this only for non production environments?

Comment (martinyonatann) [reply src/server/server.go]: sure, please review my latest commit @championswimmer

File: src/server/server.go [hunk 2]
```diff
@@ -2,13 +2,19 @@ package server
...
 	"onepixel_backend/src/routes/api"
+	"onepixel_backend/src/server/logger"
+
+	"github.com/gofiber/fiber/v2"
+	"github.com/gofiber/swagger"
+	"gorm.io/gorm"
+)
+
+const (
+	ENVIRONMENT_NAME_PRODUCTION = "production"
```
Comment (championswimmer) [inline src/server/server.go:17]: I think in Go the culture is not to use ALL_CAP_CASE for variables like Java world no?

**Review by championswimmer — APPROVED** [review]
looks good other than the const naming in CAP_CASE

File: src/server/server.go (hunk 2 above)
Comment (martinyonatann) [reply src/server/server.go:17]: Constant should use all capital letters and use underscore `_` to separate words. Eg. `INT_MAX`

Comment (martinyonatann) [reply src/server/server.go:17]: other reference https://www.cnblogs.com/zhangzhihui/articles/17881312.html

**Pull request closed without merging**